ChangeLog

```diff
@@ -14,6 +14,10 @@ Release date: TBA
 
   Closes #6074
 
+* Removed the broken ``generate-man`` option.
+
+  Closes #5283
```

Comment:
```suggestion
  Closes #5283
  Closes #1887
```

Reply:
Oh wow, this was broken for a very long time 😄

Reply:
And absolutely no one complained, it's all maintainer issues 😄